README.md

```diff
@@ -1,3 +1,23 @@
 # c360-python-client
 
 A python client for c360 projects.
+
+## Configuration
+
+This library can be configured by setting environment variables.
+
+The following configuration is mandatory:
+
+- `C360_TENANT` - the name of the tenant
+
+The following configurations are optional:
+
+- `C360_STAGE` - the target stage of the project. Can either be "prod" or "staging. Defaults to "prod"
+- `C360_API_KEY` - your API key to authenticate to the API. Can also be
+     set by calling `c360_client.dataset.set_api_key()` that will
+     prompt you to input the key
+- `C360_API_URl` - the URL to make requests to. If not given, the URL
+    will be constructed with the given `C360_TENANT` and `C360_STAGE`
+    configuration
+- `C360_BOTO_ENDPOINT_URL` - an alternate boto3 URL used to perform AWS
```

Comment:
what is an example for this ?

Reply:
When I had this, I was using `localstack` as your backend; but some parts of the library no longer support localstack.

I will remove this.

Reply:
removed in https://github.com/tra-sg/c360-python-client/pull/1/commits/51ba2f68a562397fe068e71cee51d5be5a82c7d4